perf/perf_cpu_hotplug.py

```diff
@@ -12,10 +12,12 @@
 #
 #
 # Copyright: 2021 IBM
-# Author: Nageswara R Sastry <rnsastry@linux.ibm.com>
+# Authors: Nageswara R Sastry <rnsastry@linux.ibm.com>
+#          Akanksha J N <akanksha@linux.ibm.com>
```

Comment:
I  feel we  should consider modifying author name when have more update in this module